server/src/main/java/org/elasticsearch/transport/DelegatingTransportMessageListener.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.elasticsearch.transport;
+
+import org.elasticsearch.cluster.node.DiscoveryNode;
+
+import java.util.List;
+import java.util.concurrent.CopyOnWriteArrayList;
+
+final class DelegatingTransportMessageListener implements TransportMessageListener {
```

Comment:
This is just `org.elasticsearch.transport.TransportService.DelegatingTransportMessageListener` an we should delete it from `TransportService` then?

Reply:
I deleted the stand alone class. It was from an unrelated change.